applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -143,20 +148,46 @@ Response syntax
   It indicates the MQTT broker port.
 * The ``<sec_tag>`` value is an integer.
   It indicates the credential of the security tag used for establishing a secure connection.
+* The ``<keepalive>`` value is an integer.
+  It indicates the MQTT KeepAlive value (in seconds) used in current MQTT connection.
```

Comment:
```suggestion
  It indicates the MQTT Keep Alive value (in seconds) used in the current MQTT connection.
```
I believe the feature name is "Keep Alive" with two words.

Reply:
Yes